assets/i18n/id.json

```diff
@@ -25,245 +25,243 @@
+    "dgc-getting-started-linux-5-2": "Setelah tersinkronisasi dengan jaringan, dompet Dogecoin-mu siap digunakan!",
+    "dgc-getting-started-linux-acknowledge": "Foto dan Panduan Linux dibuat oleh <a href=\"https://www.reddit.com/user/BloodyMonday\" target=\"_blank\">/u/BloodyMonday</a>. Situs Web dibuat oleh <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-android-1-1": "Menuju ke Play Store dan carilah \"Dogecoin Wallet\", atau ketuk tombol  \"Get it on Google Play\".",
+    "dgc-getting-started-android-1-2": "Tekan tombol \"Install\". Kemudian tekan tombol \"Accept\" pada jendela Permintaan Perizinan Aplikasi.",
+    "dgc-getting-started-android-1-3": "Tunggu sampai aplikasi terunduh dan terpasang.",
+    "dgc-getting-started-android-2-0": "2. Jalankan Dompetnya.",
+    "dgc-getting-started-android-2-1": "Pada halaman yang sama, tekan tombol \"Open\". Kemudian kamu akan membuka aplikasi  Dogecoin Wallet.",
+    "dgc-getting-started-android-2-2": "Kamu juga bisa membuka Dogecoin Wallet dari papan luncur di ponsel pintar Android™ kamu.",
+    "dgc-getting-started-android-3-1": "Sekarang, dompet Dogecoin ada digenggamanmu.",
+    "dgc-getting-started-android-3-2": "Di bagian atas, kamu punya Address Book dan Settings. <br/>Tepat di bawah-sebelah kiri ada alamat Dogecoinmu, lalu di sebelah kanannya ada alamat Dogecoin juga dalam bentuk kode QR, dan saldo Dogecoinmu.",
+    "dgc-getting-started-android-3-3": "Di bagian bawah, kamu bisa melihat riwayat transaksi. Trus, paling bawah, kamu bisa melakukan \"Request Coins\", \"Send Coins\", atau memindai kode QR dengan kameramu.",
+    "dgc-getting-started-android-acknowledge": "Foto dibuat oleh <a href=\"https://www.reddit.com/u/rugerty100\" target=\"_blank\">/u/rugerty100</a>. Situs Web dan Panduan dibuat oleh <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-ios-1-1": "Pada iPhone-mu, jalankan aplikasi App Store dan carilah  \"Doughwallet\", atau ketuk lencana dibawah yang akan terhubung langsung ke \"Doughwallet\". Aplikasi tersebut harusnya langsung muncul.",
+    "dgc-getting-started-ios-1-2": "Ketuk  \"Gratis\", kemudian \"Pasang\", dan masukan kata sandi iTunes ketika diminta. Aplikasi akan memulai terunduh.",
+    "dgc-getting-started-ios-2-0": "2. Tunggu sampai proses Unduh Selesai.",
+    "dgc-getting-started-ios-2-1": "Doughwallet akan membutuhkan waktu beberapa detik sampai beberapa menit untuk terunduh, tergantung kondisi kecepatan internet kamu.",
+    "dgc-getting-started-ios-2-2": "Setelah unduhan selesai, ketuk ikon untuk menjalankan aplikasi.",
+    "dgc-getting-started-ios-3-1": "Saat memulai, dompetmu akan melakukan singkronisasi sekitar 1 - 5 menit.",
+    "dgc-getting-started-ios-3-2": "Setelah proses sinkronisasi selesai, kamu siap menggunakan dompet Dogecoin-mu!",
```

Comment:
I think we can remove hypen between Dogecoin and mu

Reference: https://puebi.readthedocs.io/en/latest/kata/kata-ganti/#:~:text=Kata%20ganti%20ku%2D%20dan%20kau,Majalah%20ini%20boleh%20kaubaca.

Reply:
```suggestion
    "dgc-getting-started-ios-3-2": "Setelah proses sinkronisasi selesai, kamu siap menggunakan dompet Dogecoinmu!",
```